JapaneseFilter/sections/specific.txt

```diff
@@ -3036,6 +3036,10 @@ nc-net.or.jp##.box-pr-image-01
 mori.nc-net.or.jp###ok_ad_side
 gyutte.jp###gt_sp_all_header
 gyutte.jp###gt_sp_all_footer
+japanjournals.com###easycookieinfo
+japanjournals.com##div[class="slidebanner"]
+japanjournals.com##div[class="custom"]
```

Comment:
I think it's too broad, for example it hides link from "Hot topics" menu.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/7a3db4b1-e06f-4f60-b101-eebcc7ec881f)

</details>

Maybe something like this:
```adblock
japanjournals.com##.custom:has(> div[class^="bg_g"]> div[class^="clm"] > a[target="_blank"])
```
will be okay.